pkg/virt-handler/vm.go

```diff
@@ -3447,6 +3405,8 @@ func (c *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInsta
 		return fmt.Errorf("%s: %v", errorMessage, err)
 	}
 
+	vmi.Status.MigrationState.Completed = true
```

Comment:
I'm confused... In a previous commit, you say:
```
virt-launcher: remove Completed from MigrationMetadata
This field is no longer useful [...]
```
Why do we still use it here then?

Reply:
Leftover perhaps?

Reply:
I removed it from the migrationMetadata, I'm not removing it from the VMI Status.

This is needed to ensure we don't call finalizeMigration in an endless loop.